spec/controllers/api/v1/books_controller_spec.rb

```diff
@@ -0,0 +1,45 @@
+require 'rails_helper'
+require 'support/shared_contexts/shared_authenticated.rb'
+
+describe Api::V1::BooksController, type: :controller do
+  include_context 'Authenticated User'
+
+  describe 'GET #index' do
+    context 'When fetching all the users rents' do
+      before do
```

Comment:
Don't use a before with intance variables use `let!(:book) { create(:book) }`

Reply:
Done! I can see how 'let!' works and it's much better. Thanks.